workflow/cron/operator_test.go

```diff
@@ -713,3 +713,18 @@ func TestEvaluateWhen(t *testing.T) {
 	require.NoError(t, err)
 	assert.True(t, result)
 }
+
+func TestEvaluateWhenUnresolved(t *testing.T) {
```

Comment:
```suggestion
func TestEvaluateWhenUnresolvedOutside(t *testing.T) {
```

since this is testing "outside of `when`", might be good to clarify

Reply:
Done!